### ISSUE TITLE
How to inform algorithm owner/author that a benchmark/test failed?

### ISSUE THREAD

**Issue by soxofaan** [ESA-APEx/apex_algorithms#25]
The github based benchmark suite runs now send notification of failure to repo owners (I guess), but the actual algorithm owner/author should (also) be informed in some way

**Comment by jdries** [comment]
contact info is available in the algorithm record.
Can github email arbitrary persons?

**Comment by soxofaan** [comment]
I don't think you can tweak the recipients of the default github workflow failure notification email.
I'm afraid we would have to do the whole notification sending ourselves from our own code

**Comment by HansVRP** [comment]
Yesterday we had a meeting and came up with the following idea:

Within test_flow
- github action to create a ticket
- gitup action to post the error 
- github action to create an image showing the difference/error message
- github overwrite (update url to new url)? --> Precreate PR for updating reference?

**Comment by HansVRP** [comment]
proposal: [HansVRP/pytest-github-actions](https://github.com/HansVRP/pytest-github-actions)

**Comment by jdries** [comment]
The github action for benchmarks now automatically creates a github issue.
The issue is not yet assigned, so we need to do that manually @HansVRP  ?
